admin/package-files/upstart/flocker-agent.conf

```diff
@@ -0,0 +1,14 @@
+# flocker-agent - flocker-agent job file
+
+description "Flocker Agent service"
+author "ClusterHQ <support@clusterhq.com>"
+
+start on runlevel [2345]
+stop on runlevel [016]
+
+respawn
+
+script
+	. /etc/default/flocker-agent.conf
+	exec /usr/sbin/flocker-zfs-agent $FLOCKER_NODE_NAME $FLOCKER_CONTROL_NODE
```

Comment:
@itamarst It is still present in `master` in the systemd files.  Should it be removed from both Upstart and Systemd files?